src/functions/TestResults.ps1

```diff
@@ -966,24 +1010,17 @@ function Get-RunTimeEnvironment() {
         }
     }
 
-    if ( ($PSVersionTable.ContainsKey('PSEdition')) -and ($PSVersionTable.PSEdition -eq 'Core')) {
-        $CLrVersion = "Unknown"
-
-    }
-    else {
-        $CLrVersion = [string]$PSVersionTable.ClrVersion
-    }
-
     @{
-        'nunit-version' = '2.5.8.0'
-        'junit-version' = '4'
-        'os-version'    = $osSystemInformation.Version
-        platform        = $osSystemInformation.Name
-        cwd             = $pwd.Path
-        'machine-name'  = $computerName
-        user            = $username
-        'user-domain'   = $env:userDomain
-        'clr-version'   = $CLrVersion
+        'nunit-version'     = '2.5.8.0'
+        'junit-version'     = '4'
+        'os-version'        = $osSystemInformation.Version
+        'platform'          = $osSystemInformation.Name
+        'cwd'               = $pwd.Path
+        'machine-name'      = $computerName
+        'user'              = $username
+        'user-domain'       = $env:userDomain
+        'clr-version'       = [string][System.Environment]::Version
+        'framework-version' = [string]$ExecutionContext.SessionState.Module.Version
```

Comment:
Intentionally left this new framework-version attribute in JUnit since it adds it as generic properties.